db/connect.db.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
+require('dotenv').config();
+
+async function initialiseDbConnection() {
+  await mongoose.connect(
+    process.env['mongodb_uri'], { useUnifiedTopology: true, useNewUrlParser: true })
+    .then(console.log('Mongo DB connected'))
+    .catch(err => console.log("error in mongodb connection", err));
+}
```

Comment:
Hey,
Over here you are usingboth async await and promises. Use one of them.
I would suggest you go for async await and surround it with try catch block.

```
try{
     await mongoose.connect()
}catch(err){
  do something
}